OrbitGl/ModulesDataView.h

```diff
@@ -28,15 +28,18 @@ class ModulesDataView : public DataView {
   bool HasRefreshButton() const override { return true; }
   void OnRefreshButtonClicked() override;
 
-  void SetProcess(ProcessData* process);
+  // TODO(antonrohr) make const as soon as process_ is gone
+  void SetProcess(const ProcessData* process);
 
  protected:
   void DoSort() override;
   void DoFilter() override;
 
  private:
-  [[nodiscard]] const ModuleData* GetModule(uint32_t row) const { return modules_[indices_[row]]; }
+  [[nodiscard]] ModuleData* GetModule(uint32_t row) const { return modules_[indices_[row]]; }
 
+  // TODO(antonrohr) remove once GOrbitApp->LoadSymbols does not need process anymore
```

Comment:
That comment also seems to be outdated. If not, please rephrase or move to a different position.

Reply:
It is actually still relevant. I made it more verbose and hope it makes sense now. If not, please let me know

Reply:
Okay. Thanks!